lib/omnibus/packagers/msi.rb

```diff
@@ -73,9 +76,13 @@ class Packager::MSI < Packager::Base
             -cultures:en-us
             -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
             project-files.wixobj source.wixobj
-            -out "#{windows_safe_path(Config.package_dir, package_name)}"
+            -out "#{msi_file}"
         EOH
         shellout!(light_command, returns: [0, 204])
+
+        if cert_name
```

Comment:
I think we should probably have an option for `sign` to align with the RPM UX. Then, if that option is true, we can validate that `cert_name` is present as a validation step.